_posts/2021-05-20-wdc-persist-cookies.md

```diff
@@ -0,0 +1,15 @@
+---
+title: "Version 2.4.0 Release Notes"
+abstract: "In Tableau 2021.1.2 and later, Web Data Connectors can now enable cookie persistence across multiple WDC connections for the duration of a single Tableau Desktop session."
+---
+
+
+The WDC API has been updated to provide the function `tableau.enableCookiePersistence`. This function provides WDCs the ability to share cookies across multiple WDC connections within a single Tableau Desktop session.
+
+Once a WDC has made a call to the `tableau.enableCookiePersistence` function, any additional cookies created by WDCs in the current Tableau Desktop session will persist until the session ends.
+
+Prior to this change, if a workbook had multiple WDC connections to the same host, each WDC connection would act as an isolated web session, with isolated cookies. This meant that for a workbook with 3 WDC connections all connecting to `https://example.com` with authentication, signing in would be required 3 times, as the cookie generated from logging in on one of the connections would not be available to the other connections.
+
+Now, if a WDC triggers the `tableau.enableCookiePersistence` function, all cookies generated in WDCs will persist until the Tableau Desktop session ends. This allows WDC connections to avoid re-authenticating when all the WDC connections are authenticating on the same host.
```

Comment:
maybe "calls" instead of "triggers" here, but it's fine as is. The meaning is clear.